nixos/tests/systemd-coredump-disabled.nix

```
@@ -0,0 +1,21 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+{
+  name = "systemd-coredump-disabled";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ squalus ];
+  };
+
+  nodes.machine = { pkgs, lib, ... }: {
+    systemd.coredump.enable = false;
+    nixpkgs.config.packageOverrides = pkgs: {
+      systemd = pkgs.systemd.override {
+        withCoredump = false;
+      };
+    };
```

This makes the test cheaper:

```suggestion
    systemd.package = pkgs.systemd.override {
        withCoredump = false;
    };
```

systemd still triggers quite a few rebuilds.

---

That is indeed better. Updated.